ortools/sat/csharp/CpModel.cs

```diff
@@ -61,7 +61,7 @@ public IntVar NewConstant(long value)
 
     public IntVar NewConstant(long value, string name)
     {
-        return new IntVar(model_, new Domain(value), name);
+        return new IntVar(model_, value, name);
```

Comment:
no need for a new ctor. NewIntVar(model_,value, value, name)

Reply:
fair point 👍